proxy/server.go

```diff
@@ -9,6 +9,35 @@ import (
 	pbda "github.com/rollkit/go-da/types/pb/da"
 )
 
+// The following consts are copied from appconsts to avoid dependency hell
+const (
```

Comment:
why bring celestia consts here? this should be part of `celestia-da`.

Reply:
We need a reasonable max blob size for testing.

Reply:
Now that we're passing in this from `NewDummyDA`, the consts are part of test files, I think that's reasonable for now.